lib/step.js

```diff
@@ -112,10 +112,10 @@ class Step {
         return `${arg}`;
       } else if (arg instanceof Secret) {
         return '*****';
-      } else if (arg.toString && arg.toString() !== '[object Object]') {
-        return arg.toString();
       } else if (typeof arg === 'object') {
```

Comment:
And if arg is null, this line will return true, because null it is object ) 
And `humanizeArgs` return sting like `"null"`. 

Maybe we checks of null as first condition?

Reply:
@Vorobeyko `else if (typeof arg === 'object')` handles when arg is sent as null and `humanizeArgs` return sting `"null"` but the real issue here is that `else if (arg.toString && arg.toString() !== '[object Object]')` check comes before `else if (typeof arg === 'object')` check and breaks the condition as `arg.toString()` fails to work on `arg=null` value, so I thought of not adding any extra check and rearrange our checks a bit which will not hopefully create any side effect and handles the null condition well without having to apply any additional check. Would love to hear your feedback on this pls.

Reply:
@Vorobeyko just wanted to check if you found time to look through my comment pls

Reply:
@sjana-aj Hi, sorry for late. 
I think you must to add additionally checks on top like:
```js
if (!args) {
  return ''
}
```

And this all. 
Args never been null or undefined on line `arg.toString && arg.toString() !== '[object Object]'`